src/project-1.js

```diff
@@ -1,140 +1,86 @@
 // Do not change any of the funcxtion names
 
-const multiplyByTen = (num) => {
-  // return num after multiplying it by ten
-  // code here
-};
+const multiplyByTen = num => num * 10;
 
-const subtractFive = (num) => {
-  // return num after subtracting five
-  // code here
-};
+const subtractFive = num => num - 5;
 
 const areSameLength = (str1, str2) => {
-  // return true if the two strings have the same length
-  // otherwise return false
-  // code here
+  if (str1.length === str2.length) {
```

Comment:
This can just be 
```
return str1.length === str2.length;
```

You can also use an arrow function to simplify it: `const areSameLength = (str1, str2) => str1.length === str2.length;`